src/emotion.js

```diff
@@ -0,0 +1,23 @@
+import createEmotion from "create-emotion";
+
+const context = typeof global !== "undefined" ? global : {};
+
+if (context.__CATALOG_EMOTION_INSTANCE__ === undefined) {
```

Comment:
I was already wondering how to resolve a conflict of multiple Emotion instances – and then I saw this. Perfect.